utils/outputwriter/outputcontent.go

```diff
@@ -55,6 +74,7 @@ func getPRSummaryBanner(issuesExists, isComment bool, provider vcsutils.VcsProvi
 	return PRSummaryCommentTitleSrc(provider)
 }
 
+// TODO: remove this at the next release, it's not used anymore and replaced by adding ReviewCommentId comment to the content
```

Comment:
Please add a ticket with this task - so we won't miss it

Reply:
XRAY-46067